src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1100,8 +1100,18 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         // selection.
         if (_terminal->IsSelectionActive() && !KeyEvent::IsModifierKey(vkey) && vkey != VK_SNAPSHOT)
         {
-            _terminal->ClearSelection();
-            _renderer->TriggerSelection();
+            CoreWindow window = CoreWindow::GetForCurrentThread();

```

Comment:
nit: `window` should be `const`, can be `auto`

Reply:
Thank you, @Don-Vito!

Please advise if I should also add the const keyword to window in another line in the file: https://github.com/microsoft/terminal/blob/5633513d4f0b4e96e33b6546588cc8f5ebe417fd/src/cascadia/TerminalControl/TermControl.cpp#L2894

Reply:
I think you should - we love consts! :blush: